documentation/src/docs/deployment/kubernetes-install.md

```diff
@@ -31,8 +31,9 @@ You may obtain a copy of the License at
 
 ## Requirements
 
-- Kubernetes 1.8+
-- An existing Apache Zookeeper 3.5 cluster. This can be easily deployed using our [Zookeeper Operator](https://github.com/pravega/zookeeper-operator).
+- Kubernetes 1.15+
+- An existing Apache Zookeeper 3.6.1 cluster. This can be easily deployed using our [Zookeeper Operator](https://github.com/pravega/zookeeper-operator).
```

Comment:
Do we know the minimal version we need of Zookeeper so that Pravega can work? For instance, maybe using Zookeeper 3.5.x things work as well. It would be to have a concrete answer to that.

Reply:
According to the Zookeeper-operator docs, 3.6.1 is the minimum version